README.md

```diff
@@ -1,75 +1,87 @@
-        
-  3. After execution of Prerequisite script, Search Service Principal Name in Azure Active Directory and fetch App Id.
-  4. Onboard App Id to resource group with Contributor role.  
-  
 ## JMeter test scripts:
-  1.	create the test suite with the help of how to setup JMeter test plan(https://jmeter.apache.org/usermanual/build-web-test-plan.html).
-  2.	Check in the JMX file and supporting files in a repository
-## AKS setup 
-  1.	Create  AKS cluster with the help of how to create a AKS cluster(https://docs.microsoft.com/en-us/azure/aks/kubernetes-walkthrough-portal)
-  2.	Provide access to a Service Principal Name which would be used to run the JMX file in the cluster.
-
-## Steps to onboarding for the pipeline:
-1.	Fork the YAML pipeline from the repository:  JMeterAKSLoadTest(https://github.com/microsoft/JMeterAKSLoadTest.git)
-2.	Folder structure looks like below:
-   ![image](https://user-images.githubusercontent.com/81369583/114205274-bf9fa200-9977-11eb-9588-3185151bb711.png)
-   
-3.	Inside the JMeterFiles folder add the JMX and supporting files there
-   ![image](https://user-images.githubusercontent.com/81369583/114205337-d34b0880-9977-11eb-9b79-d728989469b0.png)
-   
-4.	Overview on the variable set up:
-  - JMX file has below variables, which can be used from the variable group or pipeline variables according to the setup:
-      1. PerfTestResourceId – Resource Id for the API Auth 
-      2.	PerfTestClientId – Client Id for the API Auth 
-      3.	PerfTestClientSecret – Client secret for the API Auth
-      4.	JmeterFolderPath – JMX File folder path
-      5.	JmeterFileName – JMX File name 
-  - AKS set up related variables:
-      1.	AKSClusterNameRegion1 -Cluster name of the respective region
-      2.	AKSResourceGroupRegion1 – Cluster resource name for the region
-      3.	AKSSPNClientIdRegion1 – client id for the region
-      4.	AKSSPNClientSecretRegion1 – client secret for the region
-      5.	TenantId – tenant id
-      6.	CSVFileNames – list of supported file names for execution like “users.csv,ids.csv”
-      
-      ![image](https://user-images.githubusercontent.com/81369583/114205527-0097b680-9978-11eb-90a4-45bd8c0a7326.png)
-5.	Set the mentioned pipeline variables as shown:
-   ![image](https://user-images.githubusercontent.com/81369583/114205558-08575b00-9978-11eb-8b1c-999b00f8e924.png)
-
-6.	Set the Variable group linked from Key vault.     
-
-7.	The results of the execution is published as artifact and it can be downloaded. The index.html file holds the report of the run.
+
+1. create the test suite with the help of how to setup JMeter test plan(https://jmeter.apache.org/usermanual/build-web-test-plan.html).
+2. Check in the JMX file and supporting files in a repository
+
+## AKS setup
```

Comment:
@Shuvajyoti , @saurabh-mridul  - Shouldn't we remove AKS setup section from here since both of our pipelines will take care of infra setup ?

Reply:
Shuva is planning to add more details under this section which will cover some details related to load-infra-pipeline